packages/calcite-components/src/components/card/card.e2e.ts

```diff
@@ -22,50 +22,59 @@ describe("calcite-card", () => {
     accessible("calcite-card");
   });
 
-  describe("accessible when selectable", () => {
+  describe("accessible when selectable (deprecated)", () => {
     accessible(
-      html`<calcite-card selectable>
+      html`<calcite-card label="example-label" selectable>
         <img slot="thumbnail" src="${placeholder}" alt="Test image" />
       </calcite-card>`,
     );
   });
 
   describe("slots", () => {
-    slots("calcite-card", SLOTS);
+    slots("calcite-card", SLOTS, true);
   });
 
   it("renders with default props if none are provided", async () => {
     const page = await newE2EPage();
     await page.setContent(`
-      <calcite-card>
+      <calcite-card label="example-label">
         <img slot="thumbnail" src="${placeholder}" alt="Test image" />
       </calcite-card>`);
 
     const element = await page.find("calcite-card");
     expect(element).not.toHaveAttribute("disabled");
     expect(element).not.toHaveAttribute("loading");
-    expect(element).not.toHaveAttribute("selectable");
     expect(element).not.toHaveAttribute("selected");
   });
 
   it("renders with requested props", async () => {
     const page = await newE2EPage();
     await page.setContent(`
-      <calcite-card loading selectable selected disabled>
+      <calcite-card label="example-label" loading selected disabled>
         <img slot="thumbnail" src="${placeholder}" alt="Test image" />
       </calcite-card>`);
 
     const element = await page.find("calcite-card");
     expect(element).toHaveAttribute("disabled");
     expect(element).toHaveAttribute("loading");
-    expect(element).toHaveAttribute("selectable");
     expect(element).toHaveAttribute("selected");
   });
 
+  it("renders with selectable (deprecated)", async () => {
```

Comment:
I think we could drop this since screenshot tests would catch this (rendering selectable affordance when enabled). Applies to similar tests. cc @brandentheintern